common/src/macros.rs

```diff
@@ -0,0 +1,12 @@
+// Licensed under the Apache-2.0 license.
+
+#[macro_export]
+macro_rules! cfi_check {
+    ($result:expr) => {
+        if cfi_launder($result.is_ok()) {
```

Comment:
I think we want to support the no-cfi flag for this macro

Reply:
I believe this is baked into the underlying `cfi_assert!` macro and `cfi_launder()` function, so we don't need to worry about it at this level.